src/admin/routes.js

```diff
@@ -1,8 +1,10 @@
-import config from 'config';
 import React from 'react';
 import { IndexRoute, Route } from 'react-router';
 
-import SimulateError from 'core/containers/SimulateError';
+import SimulateAsyncError from
+  'core/containers/error-simulation/SimulateAsyncError';
+import SimulateSyncError from
```

Comment:
In Node the variants are differentiated as well. But there the sync one has a longer name (to discourage usage?).